### PR TITLE
Email users on safety layer violation

### PR DESCRIPTION
**Pull request by SmittieC**
Email addresses to notify is specified on the experiment:

image

### CODE REVIEW

File: apps/chat/bots.py [hunk 1]
```diff
@@ -119,6 +125,8 @@ def _get_response(self, input_str: str):
         # human safety layers
         for safety_bot in self.safety_bots:
             if safety_bot.filter_human_messages() and not safety_bot.is_safe(input_str):
+                if self.session_id:
```
Comment (snopoke) [inline apps/chat/bots.py:128]: When would there be no session ID?

**Review by snopoke — COMMENTED** [review]
One question but otherwise looks good

**Review by snopoke — APPROVED** [review]

File: apps/chat/bots.py (hunk 1 above)
Comment (SmittieC) [reply apps/chat/bots.py:128]: When using the prompt builder. In the `get_prompt_builder_response_task` task we don't give the `TopicBot` a session_id

Comment (SmittieC) [reply apps/chat/bots.py:129]: Added https://github.com/dimagi/open-chat-studio/pull/190/commits/ec92e5880a825585cba03427800dd1e1b4b9ceef

**Pull request merged**